routes/account.js

```diff
@@ -41,7 +41,7 @@ function accountRoutes (server, options, next) {
         password: password,
         include: query.include,
         id: id
-      })
+      }, options, request)
```

Comment:
the [accounts API](https://github.com/hoodiehq/hoodie-account-server/tree/master/api) is somewhat independent from routes, it could even be moved to an independent module. Ideally it will match the [admin client API for accounts](https://github.com/hoodiehq/hoodie-account-client/tree/master/admin).

I would not pass in route options and the request into the `accounts.add` method, instead I’d run `hooks.account.beforeAdd` here, before running `accounts.add`. Would that work?

If I recall correctly, the database in which the user account document shall be stored, depends on the request, too? Or is that not the case any longer?